pagopa/io-app
# fix: [PE-588] Cgn card detail capitalize user name surname

## Short description
Name and surname capitalized in CGN card detail screen

## List of changes proposed in this pull request
- add capitalize function

## How to test
- Login into app
- Go to CGN card detail screen from Wallet (add it to wallet if necessary)
- The "nome" and "cognome" should be capitalized even if identity service provider provides lowercase names

<img width="358" alt="Screenshot 2024-09-11 alle 15 55 10" src="https://github.com/user-attachments/assets/d1274add-237d-4d8f-9c4b-de0ef473ff32">
